cfpb/cfgov-refresh
# Update cf-core, cf-forms, and cf-notifications to v10.2.1

These Capital Framework updates fix GHE/CFGOV/Platform/issues/2581, making the standard link hover color Dark Pacific for adequate color contrast with white and Gray 5 backgrounds. It also adds the -10 level shades to the non-Gray colors in the palette and updates the background colors of several of our notification variations to use those so that link colors have adequate contrast in those notifications.

## Changes

- cf-core updated from 10.1.0 to 10.2.1
- cf-forms updated from 10.2.0 to 10.2.1
- cf-notifications updated from 10.1.0 to 10.2.1

## Testing

1. Pull branch
1. `yarn`
1. Run your server
1. Visit http://localhost:8000/ and see that links turn to Dark Pacific on hover.
1. Visit http://localhost:8000/policy-compliance/rulemaking/regulations/ and compare the notification background color with https://www.consumerfinance.gov/policy-compliance/rulemaking/regulations/

## Checklist
